ILSpy.ReadyToRun/ReadyToRunLanguage.cs

```diff
@@ -16,35 +16,88 @@
 // OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
+// #define STRESS
```

Comment:
Is this meant to be always defined? Looks like an oversight from debugging to me.

Reply:
Nope, this is meant to be off by default. If `STRESS` is defined, the normal output will be replaced by a dummy one, and all methods will be disassembled whenever one is selected. That is useful only for testing.

Reply:
never mind, I just realized it's commented